package/test_requirements.txt

```diff
@@ -13,6 +13,8 @@ pytest~=6.2.0
 pytest-cov~=2.11.1
 pytest-mock~=3.6.1
 trufflehog~=2.1.0
+plotly~=4.14.3
+pandas~=1.3.0
```

Comment:
You don't need it here. `test_requirements` automatically uses `requirements.txt`.